testsuite/features/init_clients/centos_ssh_minion.feature

```diff
@@ -64,3 +64,7 @@ Feature: Bootstrap a SSH-managed CentOS minion and do some basic operations on i
   Scenario: Check events history for failures on SSH-managed CentOS minion
     Given I am on the Systems overview page of this "ceos_ssh_minion"
     Then I check for failed events on history event page
+
+@centos_minion
```

Comment:
* Please tag with `# WORKAROUND fix CentOS random boot failures`
* I suggest to move that commit to a separate PR if it works
* I asked Ricardo whether we should not do that at images level, rather

Reply:
It does not work, so I removed this commit.

Reply:
OK!